certinfo.go

```diff
@@ -1041,6 +1056,31 @@ func CertificateText(cert *x509.Certificate) (string, error) {
 					}
 					buf.WriteString(fmt.Sprintf("%16s%s: %s\n", "", key, value))
 				}
+			case ext.Id.Equal(oidStepManagedEndpoint):
+				buf.WriteString(fmt.Sprintf("%12sX509v3 Step Managed Endpoint:", ""))
+				if ext.Critical {
+					buf.WriteString(" critical\n")
+				} else {
+					buf.WriteString("\n")
+				}
+				val := &stepManagedEndpoint{}
+				rest, err := asn1.Unmarshal(ext.Value, val)
+				if err != nil || len(rest) > 0 {
+					return "", fmt.Errorf("certinfo: Error parsing OID %q", ext.Id.String())
+				}
+
+				// Get kind name
+				var kind string
+				if len(stepManagedEndpointKind) > val.Kind {
+					kind = stepManagedEndpointKind[val.Kind]
+				} else {
+					kind = fmt.Sprintf("%d (unknown)", val.Kind)
+				}
+
+				buf.WriteString(fmt.Sprintf("%16sKind: %s\n", "", kind))
+				if len(val.EndpointID) != 0 {
```

Comment:
What's the use case for this condition?

Reply:
If the sequence does not have an endpoint id is going to fail before.

Reply:
Ah, I previously had additional elements in the sequence, fixing this.

Reply:
[61bf429](https://github.com/smallstep/certinfo/pull/35/commits/61bf42957fdc801d9ce15e910d7f486e5ecb600a)